modules/apps/commerce/commerce-frontend-js/test/quantitySelector/utils/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import * as Utils from '../../../src/main/resources/META-INF/resources/components/quantity_selector/utils/index';
+
+describe('generateOption`s working', () => {
```

Comment:
Same thing here with the wording. This is:

```js
describe('generateOptions()', () => {
```

And the `it()` descriptions should be worded without "if" as the first word, and avoiding "should" too.